lib/LedgerEventStorage.js

```diff
@@ -517,6 +458,40 @@ class LedgerEventStorage {
     }
   }
 
+  /**
+   * Update an existing event associated with the ledger given an
+   * eventId, an array of patch instructions, and a set of options.
+   *
+   * @param eventHash - the ID of the event to update
+   * @param patch - a list of patch commands for the event
+   */
+  async updateMany({events}) {
+    const eventUpdates = events.map(({eventHash, patch}) => {
+      const {update} = _createUpdatePatch({patch});
+      return {
+        updateOne: {
+          filter: {'meta.eventHash': eventHash},
+          update
+        }
+      };
+    });
+
+    const chunks = _util.chunkDocuments(eventUpdates);
+
+    for(const chunk of chunks) {
+      const {result} = await this.collection.bulkWrite(
+        chunk, {
+          ...database.writeOptions,
```

Comment:
We can/are now configuring writeOptions at the mongodb client level and have determined that putting `writeOptions` on individual API calls is not a pattern to be continued and should be removed from use eventually.

Reply:
When the default configuration is not sufficient, what is our plan to override it?

I need to enable parallel writes to the server via `ordered: false`. I'm fine with changing the code here. What is the recommended pattern?

Reply:
I think you're just eliminating L484.

Reply:
Addressed: https://github.com/digitalbazaar/bedrock-ledger-storage-mongodb/pull/57/commits/ccd39187aa847cfc7285a2f87bdac0aa5c14129f